install/scripts/deployDynatrace.sh

```diff
@@ -33,3 +33,6 @@ echo "End applying auto tagging rules in Dynatrace "
 echo "--------------------------"
 
 ./createServiceEntry.sh $DT_TENANT_ID $DT_PAAS_TOKEN
+
```

Comment:
Does this support managed as well?

Reply:
currently not, we need to adapt a few scripts to support managed

Reply:
right. I would suggest we still merge this PR and create a new one for managed support.